internal/captain/template.go

```diff
@@ -0,0 +1,93 @@
+package captain
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"text/template"
+	"unicode"
+
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/spf13/cobra"
+)
+
+type Templater struct {
```

Comment:
I'm not sure I understand what this struct is used. None of the methods under it seems to actually work together in any meaningful way. It just seems like you're trying to save yourself passing the struct fields as arguments to the functions? 

Naming it something very generic like "Templater" also adds to the confusion.

Reply:
If was for convenience and was initially doing more than it is here. But you're right, I can just pass the arguments.